qr_web/settings.py

```diff
@@ -27,18 +26,22 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = True
 
-ALLOWED_HOSTS = []  # type: List[str]
+ALLOWED_HOSTS: typing.List = []
```

Comment:
list[str] or List[str]
1. Make type annotation simple as possible, don't call to module "typing", import from it.
2. https://typing.readthedocs.io/en/latest/source/best_practices.html to read more about typing, attentation to last header

Reply:
I disabled ALLOWED_HOSTS check for now (ALLOWED_HOSTS = ['*']), because as I understand we will includes a list of the domains supported by the project later??? Link: https://docs.djangoproject.com/en/dev/topics/testing/advanced/#topics-testing-advanced-multiple-hosts

Reply:
Later, probably, yes. But as it's not required now, don't mix your ticket with future one. When we need it, we'll add it